src/Apps/ArtQuiz/ArtQuizInterface/ArtQuizInterface.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  ArrowLeftIcon,
+  CloseIcon,
+  CSSGrid,
+  FullBleed,
+  Spinner,
+  Text,
+} from "@artsy/palette"
+import { useArtQuizContext } from "Apps/ArtQuiz/ArtQuizContext/ArtQuizContext"
+import {
+  ArtQuizDislikeButton,
+  ArtQuizSaveButton,
+} from "Apps/ArtQuiz/ArtQuizInterface/ArtQuizButtons"
+import {
+  GridClickable,
+  GridFlex,
+  GridImage,
+  GridResponsiveBox,
+  GridText,
+} from "Apps/ArtQuiz/ArtQuizInterface/ArtQuizGridComponents"
+import { useNavBarHeight } from "Components/NavBar/useNavBarHeight"
+import { FC, useEffect, useState } from "react"
+import { useWindowSize } from "Utils/Hooks/useWindowSize"
+
+const DESKTOP_ROW_HEIGHT = 120
+const DESKTOP_IMAGE_PY = 40
+const MOBILE_ROW_HEIGHT = 80
+const MOBILE_ADDRESS_BAR_HEIGHT = 55
+
+export const ArtQuizInterface: FC = () => {
+  const { desktop, mobile } = useNavBarHeight()
+  const {
+    artworksTotalCount,
+    currentArtwork,
+    currentIndex,
+  } = useArtQuizContext()
+  const { height, width } = useWindowSize()
+
+  const aspectRatio = currentArtwork.image?.aspectRatio ?? 1
+
+  const [maxHeight, setMaxHeight] = useState<number>(0)
+
+  useEffect(() => {
+    if (height && width > 768) {
+      setMaxHeight(height - DESKTOP_ROW_HEIGHT * 2 - DESKTOP_IMAGE_PY - desktop)
+    }
+    if (height && width < 768) {
+      setMaxHeight(
+        height - MOBILE_ROW_HEIGHT * 2 - MOBILE_ADDRESS_BAR_HEIGHT - mobile
+      )
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [height, width])
```

Comment:
I don't think this is necessary at all. There's nothing special about this layout that necessitates stepping out of CSS.

Reply:
Oh I think I see you're running into a problem with a responsive placeholder and maintaining that it fit vertically as well as horizontally? (Basically: don't use `ResponsiveBox`)

Reply:
The short answer here is: if you need a gray placeholder, here's some prior art I did for the article page lightbox: https://github.com/artsy/force/blob/main/src/Apps/Article/Components/ArticleZoomGallery/ArticleZoomGalleryResponsiveBox.tsx

And if you don't need a placeholder, then basically anything will work :)